src/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,94 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #TAGS
+\*------------------------------------*/
+
+/**
+ * The rounded rectangle container that houses information.
+ * 1) Grabs colors from whichever variant is selected.
+ * 2) Rounds the corners of the tag.
+ */
+.tag {
+  width: max-content;
+  max-width: 20rem;
+  padding-top: var(--eds-size-half);
+  padding-bottom: var(--eds-size-half);
+  padding-left: var(--eds-size-1-and-half);
+  padding-right: var(--eds-size-1-and-half);
+
+  display: inline-flex;
+  align-items: center;
+
+  border: var(--tag-secondary-color) solid var(--eds-theme-border-width); /* 1 */
+  border-radius: 9999px; /* 2 */
```

Comment:
Does this have the same effect as `border-radius: 50%`? Could we use `eds-border-radius-round` or something like that?

Reply:
Does not, `border-radius: 50%` creates ovals: 
![image](https://user-images.githubusercontent.com/86632227/161835481-88ae0c3e-5897-4b7d-99b3-08b18ee4ba33.png)

Reply:
Awesome, thanks for confirming 👍